packages/ui/src/components/asset-name/asset-name.module.css

```
@@ -0,0 +1,32 @@
+.root {}
+
+.notPredictive {
+  margin-right: var(--space-xxsmall);
+  display: inline-block;
+  width: var(--space-small);
+  height: var(--space-small);
+}
+
+.notPredictiveAnchor {}
```

Biome

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)